### ISSUE TITLE
Bower support

### ISSUE THREAD

**Issue by positlabs** [Timgur/gifv#6]
Would be nice to support installing via Bower

Consider using https://github.com/webcomponents/polymer-boilerplate

**Comment by anselmbradford** [comment]
Or npm? Bower's future feels shakier.

**Comment by positlabs** [comment]
Agreed, npm would be nice, too. Polymer team seems to have a preference for Bower, so that's why i suggested it over npm.
